phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -825,9 +825,25 @@ protected static void ensureTableCreated(String url, String tableName, Long ts)
 
     protected static void ensureTableCreated(String url, String tableName, byte[][] splits, Long ts) throws SQLException {
         String ddl = tableDDLMap.get(tableName);
+        createSchema(url,tableName, ts);
         createTestTable(url, ddl, splits, ts);
     }
 
+    public static void createSchema(String url, String tableName, Long ts) throws SQLException {
+        if (tableName.contains(".")) {
```

Comment:
Use SchemaUtil.getSchemaNameFromFullName and related methods here to parse the schema name and table name.